### ISSUE TITLE
[Bug]: 我插入数据成功返回id了   数据库里没有数据

### ISSUE THREAD

**Issue by healthexamination** [milvus-io/milvus#14086]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.0.0
- Deployment mode(standalone or cluster):standalone
- SDK version(e.g. pymilvus v2.0.0rc2):v2.0rc7
- OS(Ubuntu or CentOS): CentOS
- CPU/Memory: 1/2G
- GPU: 
- Others:
```

### Current Behavior

public BasicResponse<Boolean> insertEntity() {
        List<InsertParam.Field> fields = new ArrayList<>();
        List<Long> archiveIds = new ArrayList<>();
        archiveIds.add(1L);
        archiveIds.add(2L);
        List<Long> accountIds = new ArrayList<>();
        accountIds.add(1234L);
        accountIds.add(1235L);
        List<Integer> grades = new ArrayList<>();
        grades.add(4);
        grades.add(4);
        List<List<Float>> floatVectors = new ArrayList<>();
        List<Float> aaa = new ArrayList<>();
        aaa.add(Float.valueOf(2));
        aaa.add(Float.valueOf(3));
        aaa.add(Float.valueOf(3));
        aaa.add(Float.valueOf(3));
        aaa.add(Float.valueOf(3));
        aaa.add(Float.valueOf(3));
        aaa.add(Float.valueOf(3));
        List<Float> bbb = new ArrayList<>();
        bbb.add(Float.valueOf(6));
        bbb.add(Float.valueOf(7));
        bbb.add(Float.valueOf(7));
        bbb.add(Float.valueOf(7));
        bbb.add(Float.valueOf(7));
        bbb.add(Float.valueOf(7));
        bbb.add(Float.valueOf(7));
        floatVectors.add(aaa);
        floatVectors.add(bbb);
        fields.add(new InsertParam.Field("archive_id", DataType.Int64, archiveIds));
        fields.add(new InsertParam.Field("account_id", DataType.Int64, accountIds));
        fields.add(new InsertParam.Field("grade", DataType.Int32, grades));
        fields.add(new InsertParam.Field("user_feature", DataType.FloatVector, floatVectors));
        InsertParam insertParam = InsertParam.newBuilder()
                .withCollectionName(collectionName)
                .withPartitionName("user_data")
                .withFields(fields)
                .build();
        R<MutationResult> insert = milvusServiceClient.insert(insertParam);
        return BasicResponse.success();
    }

### Expected Behavior

![c25be7f378675fc01a23321aadcdbac](https://user-images.githubusercontent.com/48131803/147223098-a312ee47-e54b-40d7-ab3e-834e75008090.png)

**Comment by ss892714028** [comment]
Hi, would you please elaborate on your question? Did you do a search operation? Would you please also attach any error msg if there's any?

**Comment by healthexamination** [comment]
> Hi, would you please elaborate on your question? Did you do a search operation? How did you confirm that there are no vectors stored in Milvus after the insert? Would you please also attach any error msg if there's any?

![image](https://user-images.githubusercontent.com/48131803/147311357-fbbe7ba1-f904-4425-a89a-1cfc4f02853d.png)
I see this page after i insert data.  but i can't see anything

**Comment by healthexamination** [comment]
> Hi, would you please elaborate on your question? Did you do a search operation? How did you confirm that there are no vectors stored in Milvus after the insert? Would you please also attach any error msg if there's any?
![image](https://user-images.githubusercontent.com/48131803/147311710-6dc6bfb8-e02f-4ea1-908b-40b534654b9e.png)

this is my serach operation result,  means no data?

**Comment by yhmo** [comment]
@healthexamination 
Recently we have released a stable version 2.0GA, which fixes a lot of critical issues. Use this version to avoid the weird bug.
